kaldb/pom.xml

```diff
@@ -19,8 +19,7 @@
         <armeria.version>1.16.0</armeria.version>
         <micrometer.version>1.8.5</micrometer.version>
         <kafka.version>2.5.1</kafka.version>
-        <jackson.version>2.10.5</jackson.version>
-        <jackson.databind.version>${jackson.version}.1</jackson.databind.version>
```

Comment:
jackson databind exists for 2.13.3

Reply:
We may want to leave this in fwiw - I looked into this a bit more after adding the initial comment, and it looks like the databind version pretty frequently has point releases past the major Jackson version.